doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -280,6 +280,20 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-4-1 v2-4-0
+
+KRKNWK-17699: Non complaiant Basic Information cluster revision for Matter 1.1.0. The issue result in fail at TC-BINFO-1.1 Matter Certifiction test.
+
+  **Workaround:** Set the default value of the Basic Information cluster revision to 1 in ZAP and regenerate files.
```

Comment:
I think we should be more explicit here: 
- point directly to the file which needs to be updated
- mention how the files can be regenerated (I guess saying "with ZAP Tool" would suffice)